mods/d2k/rules/aircraft.yaml

```diff
@@ -72,16 +49,17 @@ frigate:
 		Name: Frigate
 		Description: Supply spacecraft
 	Plane:
-		ROT: 5
-		Speed: 350
-		RepairBuildings: repair
+		CruiseAltitude: 1800
+		ROT: 7
+		Speed: 296	##148
```

Comment:
Disagree with what? You don't want the mod at middle D2k game speed, it's ludicrously slow and no one (especially in multiplayer) plays it at that speed.